server/src/BitBakeProjectScanner.ts

```diff
@@ -196,8 +195,15 @@ export class BitBakeProjectScanner {
 
     for (const layer of this._layers) {
       try {
-        const files = find.fileSync(new RegExp(`.${pattern}$`), layer.path)
-        for (const file of files) {
+        let matchedFiles: string[] = []
+        fs.readdir(layer.path, (err, files) => {
```

Comment:
It does not work the same. On poky I obtain the following result:
```
info: ******************************************************************
info: Layer:     3
info: Recipes:   843
info: Inc-Files: 0
info: bbclass:   0
```
While I was obtaining this result before:
```
info: ******************************************************************
info: Layer:     3
info: Recipes:   843
info: Inc-Files: 313
info: bbclass:   208
```

I think the issue is readdir is not going into the subfolders. I tried with `fs.readdir(layer.path, { recursive: true } (err, files)`, but then I obtain an other mysterious error about `filter` being not callable.

Reply:
ok, I think I will put back `find` and let's see if there is a better replacement in the future. Since this module is only used here.